packages/scripts/config/webpack.config.js

```diff
@@ -64,11 +64,11 @@ const config = {
 			mode === 'production' && ! process.env.WP_BUNDLE_ANALYZER,
 		splitChunks: {
 			cacheGroups: {
-				styles: {
-					name: 'style',
+				style: {
 					test: /style\.(sc|sa|c)ss$/,
```

Comment:
```suggestion
					test: /[\\/]style\.(sc|sa|c)ss$/,
```

This should be enough to limit it to only `style.*` files.

Reply:
Let’s test it 😃